pythonPackages.nix

```diff
@@ -47,6 +47,18 @@ let
       rev = "9b60184c5d161e4871c91ce29a44e3ac2c2a438e";
       sha256 = "1vh8dp3nw4fk1mnfv0w8ici1lzxyfn5han7hipqzsfxl75w76r18";
     };
+
+    pysisyphus = callPackage ./pkgs/apps/pysisyphus {
+      orca = selfPkgs.orca;
+      turbomole = selfPkgs.turbomole;
+      cfour = selfPkgs.cfour;
+      # gaussian = if cfg.optpath != null then self.gaussian else null;
+      # molpro = if cfg.licMolpro != null then self.molpro else null;
```

Comment:
`molpro` is now `null` if no license key is provided. We can drop the conditionals checking for `cfg.licMolpro`.

Reply:
fixed for molpro but `gaussian` is still an unknown attribute to `selfPkgs`. This is simply missing if `optpath` is not set, right?

Reply:
I've did the same to gaussian as you did to molpro as a workaround :slightly_smiling_face: